.c9/runners/C (make).run

```diff
@@ -8,5 +8,6 @@
 
     "info" : "Started $project_path: $file_name\nFile path: $file_path\nFile: $file\nArguments: $args\nFile name: $file_name\nFile extension: $file_extension\nFile base name: $file_base_name\nPackages: $packages\nProject path: $project_path\nProject name: $project_name\nHostname: $hostname\nHostname path: $hostname_path\nURL: $url\nPort: $port\nIP: $ip\nCommand: $command\nPython: $python\nPython path: $python_path",
     "env" : {},
-    "selector" : "source.c"
+    // "selector" : "source.c"
```

Comment:
Let's just delete it. I think version control is our friend here.

Reply:
Sounds good.